static/js/util/profile_edit.js

```diff
@@ -46,11 +46,11 @@ export function boundRadioGroupField(keySet, label, options) {
   const radioButtons = options.map(obj => {
     let helper = "";
     if (obj.helper) {
-      helper = `- ${obj.helper}`;
+      helper = `${obj.helper}`;
     }
     let label = (
       <span className="radio-label">
-        {obj.label}<span className="radio-label-hint">{helper}</span>
+        {obj.label}<p className="radio-label-hint">{helper}</p>
```

Comment:
Sorry, I wasn't clear. 

Instead of 

```
    if (obj.helper) {
      helper = `${obj.helper}`;
    }
    let label = (
      <span className="radio-label">
        {obj.label}<p className="radio-label-hint">{helper}</p>
      </span>
    );
```	

can we just do:

```
    let label = (
      <span className="radio-label">
        {obj.label}
        <p className="radio-label-hint">{obj.helper}</p>
      </span>
    );

```